app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -399,15 +402,26 @@ class KiwixMainActivity : CoreMainActivity() {
       title = getString(string.menu_support_kiwix),
       iconRes = drawable.ic_support_24px,
       visible = true,
-      onClick = { openSupportKiwixExternalLink() },
+      onClick = { openExternalLinkWithDialog(KIWIX_SUPPORT_URL) },
       testingTag = LEFT_DRAWER_SUPPORT_ITEM_TESTING_TAG
     )
   }
 
   /**
    * In kiwix app we are not showing the "About app" item so returning null.
    */
-  override val aboutAppDrawerMenuItem: DrawerMenuItem? = null
+  override val aboutAppDrawerMenuItem: DrawerMenuItem? by lazy {
```

Comment:
This menu item is disabled for the `Kiwix` app. The comment is written above it. Why are we enabling it?

This menu item is configurable for custom apps. The logic should be placed there.

https://github.com/kiwix/kiwix-android/blob/4e0cea3c8cf58e62b1230ce9ae10b0ad45f5972b/custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomMainActivity.kt#L134